components/language_filter/fondant_component.yaml

```
@@ -0,0 +1,14 @@
+name: Language filter
+description: A component that filter a provided dataframe based on the language.
+image: ghcr.io/ml6team/language_filter:latest
```

Nit: let's use consistent naming for all components, I'm currently using verb + noun, so this one can be named "Filter language"
```suggestion
name: Filter languages
description: A component that filters text based on the language.
image: ghcr.io/ml6team/filter_language:latest
```